src/linear_algebra/basic.lean

```diff
@@ -1112,6 +1112,9 @@ variables {f : M →ₛₗ[τ₁₂] M₂}
 include R
 open submodule
 
+lemma ker_subgroup_eq_group_hom_ker (f : M →ₛₗ[τ₁₂] M₂) :
```

Comment:
```suggestion
lemma to_add_subgroup_ker (f : M →ₛₗ[τ₁₂] M₂) :
```

Reply:
maybe check around the file to see whether we prefer this as a prefix or suffix.

The `mker` version is worth having too.

Reply:
I think `ker_to_add_subgroup` is clearer